src/utils/astropy_util.py

```diff
@@ -0,0 +1,30 @@
+from astropy.time import Time
+from astropy.coordinates import get_sun, get_moon, CartesianRepresentation
+from astropy.constants import G, M_sun, M_earth
+from typing import Tuple, Optional
+from utils.constants import BodyEnum
+from astropy.coordinates import SkyCoord
+
+
+# Heavily reference the get_ephemeris function here: https://github.com/Cislunar-Explorers/FlightSoftware/blob/master/OpticalNavigation/core/observe_functions.py
+def get_ephemeris(observeStart: float, body: BodyEnum) -> Tuple[float, float, float]:
+    # Astropy needs unix timestamp in seconds!!!
+    current_time = observeStart
+    init_au = SkyCoord(0, 0, 0, unit="m", frame="gcrs", representation_type="cartesian")
```

Comment:
Is this some initialization thing?

Reply:
Yeah, pyright was giving us issues when we set current_au to None because it feared the case where we'd try to do None.x, None.y, etc...so we set it to a default value instead.